doc/includes/API.md

```diff
@@ -3955,13 +3955,24 @@ Type|Description
 const sql = queryBuilder.toString();
 ```
 
-Returns the SQL string. If this query builder executes multiple queries, only the first query's SQL is returned.
+Returns the SQL string suitable for logging input _but not for execution_, via Knex as `this.build().toString()`. This method should not be used to create queries for database execution because it makes no guarantees about escaping bindings properly.
+
```

Comment:
Unfortunately `this.build()` is a private method and shouldn't be mentioned here as an option,